SpatialGDK/Documentation/content/glossary.md

```diff
@@ -1,33 +1,119 @@
 # Glossary
```

Comment:
Can we add a TOC, using an Improbadoc macro? I believe it wouldn't render properly on GitHub but it would be really useful for those viewing the page on the main docs site.

`<%(TOC)%>`